saos-batch/src/main/java/pl/edu/icm/saos/batch/core/TagPostUploadJobConfiguration.java

```diff
@@ -78,6 +90,7 @@ public Job tagPostUploadJob() {
         return jobs.get("TAG_POST_UPLOAD_PROCESSING")
                 .start(updateEnrichmentHashStep())
                 .next(markChangedTagJudgmentsAsNotIndexedStep())
+                .next(saveEnrichmentTagsLawJournalEntriesStep())
```

Comment:
I would remove the 's' in the middle -> saveEnrichmentTag(s)LawJournalEntriesStep